chall02/ccardozo.py

```diff
@@ -0,0 +1,59 @@
+# **************************************************************************** #
```

Comment:
A shebang is missing, it's important to use it, you can find a lot of documentation or debate about how to write it 😉